CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+# v2.3.2 (2024-08-13)
+
+## OS Changes
+
+* Kernel 6.1.102-108.177 ([#73])
+
+## Orchestrator Changes
+
+### Kubernetes
+
+* Update ecr-credential-provider to latest upstream ([#66])
+
+## Build Changes
+
+* Re-enable fmt and licenses lints in CI ([#69])
+* Use workspace dependencies for all dependencies ([#70])
+
+[#66]: https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/66
```

Comment:
We're missing mention of the soci package update that was included in this PR

Reply:
Fair: it is included in our aws-dev variant and of course OOTB can include it.